### ISSUE TITLE
thesis problem, multiple tex files and a few figures / pdf (appendix) + biblio

### ISSUE THREAD

**Issue by raphaellegiguere** [am009/git-latexdiff-web#2]
Hello! The tool seems very cool and useful but I can't get it to work. I did my thesis on Overleaf and I have some corrections/ changes to do. The figures seems to work OK but the bibliography and the "\citep{}" inside my thesis don't work (they come up as ??? on the file). Is there a work around ? I really need the bibliography included ?
Thanks!

**Comment by am009** [comment]
The bibliography is not shown by default. Did you select "bibtex" or "biber" in Settings - Bibliography?

**Comment by raphaellegiguere** [comment]
Yes. Both of them dont work. It just crashes whenever I check bibtex or biber. I used natbib and the « \citep{} » format. I also tried chantier the citep for just cite but still doesnt work.

**Comment by am009** [comment]
Sorry for the late reply, is there any error message? Or provide a sample tex project that can reproduce this problem?
